test/e2e/bootstrap_resources.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Declares the structure of the bootstrapped resources and provides a loader
+for them.
+"""
+
+from dataclasses import dataclass
+from acktest.bootstrapping import Resources
+from acktest.bootstrapping.iam import Role
+from acktest.bootstrapping.vpc import VPC
+from e2e import bootstrap_directory
+
+@dataclass
+class BootstrapResources(Resources):
+    ClusterVPC: VPC
+    ClusterRole: Role
+    FargatePodRole: Role
+    NodegroupRole: Role
```

Comment:
Also copy pasted from EKS controller. Add the resources needed for ACMPCA test setup here

Reply:
+1